articles/getting-started-with-ui-layouts-in-android/index.md

```diff
@@ -0,0 +1,140 @@
+**Note: One can only use one root element for the UI of an application.**
+- View: This refers to an object in Android's built-in View class, represented by the rectangular area of the screen, e.g., `TextView`, `ImageView`, `Button`, and others.
+- View Group: This refers to an 'invisible container' that holds multiple views or view groups together, e.g. Linear Layout, Grid Layout, Table Layout, and others.
+
+![Anatomy of Android Layouts](/engineering-education/getting-started-with-android-ui-layouts/anatomy.jpg)
+
+### Root element layouts used in android applications
+### A) Linear Layout
+This layout aligns its content in a single direction. It can be either vertical or horizontal. The layout uses the `<LinearLayout>` tag.
+One can use it either as a root element or a view group.
+
+![Linear Layout](/engineering-education/getting-started-with-android-ui-layouts/linear.jpg)
+
+#### Attributes
+The following attributes are specific to Linear Layout:
+- `android:id` - This is a unique ID signed to a specific layout.
+- `android:baselineAligned` – This is a Boolean value ( "true" or "false" ) that prevents the layout from aligning its children's baselines.
+- `android:gravity` – This specifies an object's position on both the vertical and horizontal axes.
+- `andoid:orientation` – This specifies the vertical or horizontal arrangement of objects.
+- `android:weightSum`- Sum up of child weight.
+- `android:baselineAlignedChildIndex`- It specifies which of its children to baseline align.
+- `android:divider` - This refers to a drawable used as a vertical divider between buttons.
+
+#### Pros and Cons
+#### Pros
+- The linear layout is relatively easy to use.
+- It has a weight property that ensures logical spacing.
+- Predefined width and height, either match parent (Covers the entire screen space) or wrap content (covers the content added to the screen).
+#### Cons
+- Has basic functionality hence making it difficult to use when creating a complex UI.
+- If the UI is too complex during nesting, the system run time is not completed within the allowed 16ms, which affects the UX(User Experience) of an application and its quality.
+
+### B) Relative Layout
+This layout positions views relative to each other. The layout uses the `<RelativeLayout>` tag.
+The position of each view can be relative to sibling elements or relative to the parent. By default, all child views are aligned at the top corner before setting the screen's desired alignment, for example, center-left and others. One can use it either as a root element or a view group.
+
+![Relative Layout](/engineering-education/getting-started-with-android-ui-layouts/relative.jpg)
+
+#### Attributes
+The following attributes are specific to Relative Layout:
+- `android:id` - This is a unique ID signed to a specific layout.
+- `android:gravity` – This specifies an object's position on both the vertical and horizontal axes.
```

Comment:
Id and gravity are not specific to relative layout. I'd suggest you omit the first sentence or remove the attributes that are not specific to relative layout.